refinery/core/serializers.py

```diff
@@ -128,6 +129,11 @@ class ExtendedGroupSerializer(serializers.ModelSerializer):
         validators=[UniqueValidator(queryset=ExtendedGroup.objects.all())]
     )
 
+    def get_manager_group_uuid(self, group):
```

Comment:
Seems redundant. Doesn't appear to be used anywhere.

Reply:
It's a field the client expects.